confgenerator/confgenerator.go

```diff
@@ -124,12 +124,15 @@ func (uc *UnifiedConfig) GenerateOtelConfig(hostInfo *host.InfoStat) (string, er
 		uc.Metrics.Service.LogLevel = "info"
 	}
 	otelConfig, err := otel.ModularConfig{
-		LogLevel:                        uc.Metrics.Service.LogLevel,
-		ReceiverPipelines:               receiverPipelines,
-		Pipelines:                       pipelines,
-		GlobalProcessors:                []otel.Component{gcpResourceDetector()},
-		GoogleCloudExporter:             googleCloudExporter(userAgent),
-		GoogleManagedPrometheusExporter: googleManagedPrometheusExporter(userAgent),
+		LogLevel:          uc.Metrics.Service.LogLevel,
+		ReceiverPipelines: receiverPipelines,
+		Pipelines:         pipelines,
+		GlobalProcessors:  []otel.Component{gcpResourceDetector()},
+		Exporters: map[otel.ExporterType]otel.Component{
+			otel.System: googleCloudExporter(userAgent, false),
+			otel.OTel:   googleCloudExporter(userAgent, true),
```

Comment:
quick question: For existing 3P metrics, when users upgrade to Ops Agent with this change in, they will see a discontinuity in their time series at the time of upgrade right? From before and after the label was added.

because this is a static valued label, it won't disaggregate the existing metrics so won't "break" any alerts or dashboards. But are there other unintended affects to a user that we might care about? Do we need some public docs about this change?

Reply:
Yeah, they'll see a discontinuity. This is in the set of acceptable changes because it won't change the behavior of any alerts or dashboards. We have b/268237708 to document the new fields.